contrib/anubis/botPolicy.yaml

```
@@ -0,0 +1,374 @@
+    path_regex: ^/assets.*$
+    action: ALLOW
+
+  # might be used by our bots and nobody knows this URL anyways
+  - name: sc-route
+    path_regex: ^/snapshot-changes.*$
+    action: ALLOW
+
+  - name: nginx-status-route
+    path_regex: ^/nginx_status.*$
+    action: ALLOW
+
+  - name: munin-route
+    path_regex: ^/munin.*$
+    action: ALLOW
+
+  # Requires login anyways
+  - name: admin-routes
+    path_regex: ^/admin/.*$
+    action: ALLOW
+
+  # Won't work otherwise
+  - name: health-route
+    path_regex: ^/health.*$
+    action: ALLOW
+
+  # This has some irony
+  - name: allow-our-own-ai
+    path_regex: ^/experimental/mcp.*$
+    action: ALLOW
+
+  # For now AI bots are not good at JS, so allow AJAX
+  - name: allow-ajax
+    path_regex: ^.*ajax.*$
+    action: ALLOW
+
+  - name: allow-ws
+    path_regex: ^.*ws-console.*$
+    action: ALLOW
+
+  - name: allow-ws2
+    path_regex: ^.*liveviewhandler.*$
+    action: ALLOW
+
+
+dnsbl: false
+
+# #
+# impressum:
+#   # Displayed at the bottom of every page rendered by Anubis.
+#   footer: >-
+#     This website is hosted by Zombocom. If you have any complaints or notes
+#     about the service, please contact
+#     <a href="mailto:contact@domainhere.example">contact@domainhere.example</a>
+#     and we will assist you as soon as possible.
+
+#   # The imprint page that will be linked to at the footer of every Anubis page.
+#   page:
+#     # The HTML <title> of the page
+#     title: Imprint and Privacy Policy
+#     # The HTML contents of the page. The exact contents of this page can
+#     # and will vary by locale. Please consult with a lawyer if you are not
+#     # sure what to put here
+#     body: >-
+#       <p>Last updated: June 2025</p>
+
+#       <h2>Information that is gathered from visitors</h2>
+
+#       <p>In common with other websites, log files are stored on the web server saving details such as the visitor's I
+P address, browser type, referring page and time of visit.</p>
```

this one is weird. It's not commented. So I guess this is either silently ignored or causes some warnings